roboteam_ai/src/interface/widget.cpp

```diff
@@ -29,6 +29,8 @@ void Visualizer::paintEvent(QPaintEvent* event) {
         if (showPath) {
             for (auto robot : selectedRobots) {
                 drawDataPoints(painter, Drawer::getGoToPosLuThPoints(robot.id));
+                drawDataPoints(painter, Drawer::getKeeperPoints(robot.id),7);
```

Comment:
what is this 7?

Reply:
Length of the vector of points used for drawing. This is the function you used but it was implicit on the line above (can't do that with my case because of accessing)